### PR TITLE
PMM-11699 Reviewed and updated the advisor checks for PostgreSQL

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: docs/advisors/checks/configuration-pg-check-fsync-enabled.md [hunk 1]
```diff
@@ -1,33 +1,40 @@
 # fsync is disabled on PostgreSQL configuration
 
 ## Description
-When this parameter is enabled (which it is by default), the PostgreSQL server tries to make sures that updates are physically written to disk, by issuing fsync() system calls or various equivalent methods mentioned in wal_sync_method. 
+By default, `fsync` is enabled in PostgreSQL so that the PostgreSQL server tries to make sure that updates are physically written to disk by issuing `fsync()` system calls or various equivalent methods mentioned in `wal_sync_method`.
```
Comment (elchinoo) [inline docs/advisors/checks/configuration-pg-check-fsync-enabled.md:4]: Wouldn't be better to simplify with something like:

_Fsync` is enabled by default in PostgreSQL_ ...

File: docs/advisors/checks/configuration-pg-check-fsync-enabled.md [hunk 2]
```diff
@@ -1,33 +1,40 @@
 # fsync is disabled on PostgreSQL configuration
 
 ## Description
-When this parameter is enabled (which it is by default), the PostgreSQL server tries to make sures that updates are physically written to disk, by issuing fsync() system calls or various equivalent methods mentioned in wal_sync_method. 
+By default, `fsync` is enabled in PostgreSQL so that the PostgreSQL server tries to make sure that updates are physically written to disk by issuing `fsync()` system calls or various equivalent methods mentioned in `wal_sync_method`. 
 
+Turning off `fsync` can slightly improve performance with the risk of unrecoverable data corruption in case of a power failure or system crash. 
 
-Turning off fsync can slightly improve performance with the risk of unrecoverable data corruption in case of a power failure or system crash. 
+Therefore, it is only advisable to turn off `fsync` if you can easily recreate your entire database from external data.
```
Comment (elchinoo) [inline docs/advisors/checks/configuration-pg-check-fsync-enabled.md:8]: I would rephrase here to emphasize even more that it isn't advisable, for example:

_It's not advisable to turn off `fsync` unless the database can be easily recreated from an external data source._

File: docs/advisors/checks/performance-pg-archiver-failing.md [hunk 3]
```diff
@@ -15,16 +15,16 @@ archive_command = 'copy "%p" "C:\\server\\archivedir\\%f"'  # Windows
...
 
 
 ## Resolution
 
 From the PostgreSQL log we should be able to get insight about the failure, we can even try the command manually to verify the result. 
 
-Something to keep in mind is the archive command will be executed under the ownership of the same user that the PostgreSQL server is running as, so the paths, tools, scripts, etc., that are called or acceded from the archive command should be accessible for the user running the service (usually postgres).
+Something to keep in mind is that the same user that runs the PostgreSQL server (usually postgres), will execute the archive command, so the paths, tools, scripts, etc., that are called or acceded from the archive command should be accessible for the user running the service.
```
Comment (elchinoo) [inline docs/advisors/checks/performance-pg-archiver-failing.md:25]: I'm not sure if "**acceded**" would be the best wording here. I couldn't easily understand what it intends to say in this context. We may simplify a bit? We should remember that many no-native speakers will use this documentation, and using simpler daily vocabulary is a must.

**Review by catalinaadam — APPROVED** [review]
